dev/com.ibm.ws.ejbcontainer.core/resources/com/ibm/ejs/container/container.nlsprops

```diff
@@ -1561,6 +1561,11 @@ NAMESPACE_IN_LOCAL_JNDI_BINDING_NAME_CNTR0340W=CNTR0340W: The {2} Java Naming an
 NAMESPACE_IN_LOCAL_JNDI_BINDING_NAME_CNTR0340W.explanation=The local binding name can only contain one namespace and it must start with ejblocal:. Namespaces are denoted by the colon (:) character. The entire string before and including the colon (:) character is considered a namespace.
 NAMESPACE_IN_LOCAL_JNDI_BINDING_NAME_CNTR0340W.useraction=Correct the ibm-ejb-jar-bnd.xml bindings file to give this bean a valid JNDI binding name by removing the namespace that is not valid.
 
+#-------------------------------------------------------------------------------
+UNABLE_TO_BIND_REMOTE_BINDING_CNTR0341W=CNTR0341W: Unable to bind the {0} interface of the {1} bean in the {2} module of the {3} application to the {4} name location. The CORBA name server is not available.
+UNABLE_TO_BIND_REMOTE_BINDING_CNTR0341W.explanation=The remote enterprise bean binding cannot be created because the CORBA name server is not available.
+UNABLE_TO_BIND_REMOTE_BINDING_CNTR0341W.useraction=Check the log file for exceptions that are related to the ORB or CORBA name server initialization.
```

Comment:
As far as abbreviations go- would the log messages use ORB or COBRA ? If so, no need to spell them both out, as that would make the messages longer and less legible.

I'm assuming there is a reasonable expectation that someone reading this message is familiar with the COBRA name server, if that's also the case with ORB, no need to spell out. ORB is a little more difficult here because we dont say what kind of entity it is. 

Generally, software entities should have a noun descriptor after them, eg "the xyz.example _property_" or " the COBRA _name server_.  " But I saw Tracy's recommendation not to use _service_ with ORB in this context.

Reply:
In the failure case that spurred this pull request, all they would have gotten was a FFDC stack coming from CORBA:

```
Exception = org.omg.CORBA.INITIALIZE
Source = com.ibm.ws.transport.iiop.internal.ORBWrapperInternal
probeid = 108
Stack Dump = org.omg.CORBA.INITIALIZE: :  vmcid: 0x0 minor code: 0x0  completed: No
	at org.apache.yoko.orb.spi.naming.NameServiceInitializer.post_init(NameServiceInitializer.java:143)
	at org.apache.yoko.orb.OBCORBA.ORB_impl.initialize(ORB_impl.java:460)
	at org.apache.yoko.orb.OBCORBA.ORB_impl.setParameters(ORB_impl.java:958)
	at org.apache.yoko.orb.OBCORBA.ORB_impl.set_parameters(ORB_impl.java:1300)
	at org.omg.CORBA.ORB.init(ORB.java:179)
	at com.ibm.ws.transport.iiop.yoko.ORBConfigAdapter.createORB(ORBConfigAdapter.java:78)
	at com.ibm.ws.transport.iiop.yoko.ORBConfigAdapter.createServerORB(ORBConfigAdapter.java:53)
	at com.ibm.ws.transport.iiop.internal.ORBWrapperInternal.activate(ORBWrapperInternal.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	Caused by: org.omg.CORBA.COMM_FAILURE: bind() failed: EDC8115I Address already in use. (Bind failed):  vmcid: Apache minor code: 0x8  completed: No
	at org.apache.yoko.orb.OCI.IIOP.CommFailures$$Lambda$39/0x0000000005230430.apply(Unknown Source)
	at org.apache.yoko.util.Exceptions.as(Exceptions.java:48)
	at org.apache.yoko.orb.OCI.IIOP.CommFailures.wrap(Exceptions.java:58)
	at org.apache.yoko.orb.OCI.IIOP.CommFailures.wrap(Exceptions.java:32)
	at org.apache.yoko.logging.VerboseLogging.wrapped(VerboseLogging.java:149)
	at org.apache.yoko.orb.OCI.IIOP.Acceptor_impl.<init>(Acceptor_impl.java:318)
	at org.apache.yoko.orb.OCI.IIOP.AccFactory_impl.create_acceptor(AccFactory_impl.java:187)
	at org.apache.yoko.orb.OBPortableServer.POAManagerFactory_impl.create_POAManager(POAManagerFactory_impl.java:213)
	at org.apache.yoko.orb.OBPortableServer.POAManagerFactory_impl.create_POAManager(POAManagerFactory_impl.java:74)
	at org.apache.yoko.orb.OB.ORBControl.initializeRootPOA(ORBControl.java:495)
	at org.apache.yoko.orb.OBCORBA.ORB_impl.resolve_initial_references(ORB_impl.java:986)
	at org.apache.yoko.orb.OBPortableInterceptor.ORBInitInfo_impl.resolve_initial_references(ORBInitInfo_impl.java:130)
	at org.apache.yoko.orb.spi.naming.NameServiceInitializer.post_init(NameServiceInitializer.java:101)
	... 188 more
	Caused by: java.net.BindException: EDC8115I Address already in use. (Bind failed)
	at java.net.PlainSocketImpl.socketBind(Native Method)
	at java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:417)
	at java.net.ServerSocket.bind(ServerSocket.java:444)
	at com.ibm.ws.transport.iiop.yoko.helper.SocketFactoryHelper.attemptSocketBind(SocketFactoryHelper.java:139)
	at com.ibm.ws.transport.iiop.yoko.helper.SocketFactoryHelper.openSocket(SocketFactoryHelper.java:68)
	at com.ibm.ws.transport.iiop.yoko.DefaultSocketFactory.createServerSocket(DefaultSocketFactory.java:65)
	at org.apache.yoko.orb.OCI.IIOP.Acceptor_impl.<init>(Acceptor_impl.java:307)
	... 195 more
```

Reply:
I had recommended changing "ORB service" to "ORB", because I didn't see it referenced in other messages or documentation in Liberty as a "service".  "service" seemed like an implementation detail.

However, I understand that it might make for better translations... and I also see that WebSphere traditional does have documentation that refers to it as the "ORB service", so I would be fine if we added that back into the message.

Reply:
Looking at the tWAS doc, I do see "ORB service", but also "ORB listener" and just "ORB". If this message doesn't relate specifically to an ORB service or listener, I think it's ok to just use ORB, especially since `.orb` shows up in the stack trace. I'll approve as is but if you need to make a change I can re-approve.